src/sql/parts/modelComponents/inputbox.component.ts

```diff
@@ -152,7 +152,7 @@ export default class InputBoxComponent extends ComponentBase implements ICompone
 	/// IComponent implementation
 
 	public layout(): void {
-		this._changeRef.detectChanges();
+		super.layout();
 		this.layoutInputBox();
```

Comment:
Dumb question: shouldn't we detect changes after updating width & height? E.g. call super.layout 2nd?

Reply:
I'm not sure but I think calling the method here is not necessary because I can see it's called when properties are set  which is before calling layout. but I don't want to change it now. I'll add a comment

---
In reply to: [214411421](https://github.com/Microsoft/sqlopsstudio/pull/2351#discussion_r214411421) [](ancestors = 214411421)